app/laravel/.env.test

```
@@ -29,7 +29,7 @@ REDIS_PASSWORD=null
 REDIS_PORT=6379
 
 MAIL_MAILER=smtp
-MAIL_HOST=mailhog
+MAIL_HOST=127.0.0.1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MAIL_HOST key should go before the MAIL_MAILER key

(UnorderedKey)